### PR TITLE
[skip actions] update gemspec for rebranding

### PR DESCRIPTION
**Pull request by xuan-cao-swi**

### CODE REVIEW

**Review by tammy-baylis-swi — APPROVED** [review]
Lgtm!

File: solarwinds_apm.gemspec [hunk 1]
```diff
@@ -8,7 +8,7 @@ Gem::Specification.new do |s|
 
   s.license = "Apache-2.0"
 
-  s.authors = ["Maia Engeli", "Peter Giacomo Lombardo", "Spiros Eliopoulos"]
+  s.authors = ["Maia Engeli", "Peter Giacomo Lombardo", "Spiros Eliopoulos", "Xuan Cao"]
   s.email = %q{technicalsupport@solarwinds.com}
   s.homepage = %q{https://cloud.solarwinds.com/}
```
Comment (cheempz) [inline solarwinds_apm.gemspec:13]: Hmm, homepage link doesn't work and after discussion with Erica let's use a placeholder `https://documentation.solarwinds.com/en/success_center/observability/content/intro/landing-page.htm` which will redirect to the future observability product overview page.

Same for line 22 below too :)

**Review by cheempz — COMMENTED** [review]
LGTM, thanks @xuan-cao-swi! Minor change please, let's use a placeholder for homepage.

File: solarwinds_apm.gemspec (hunk 1 above)
Comment (xuan-cao-swi) [reply solarwinds_apm.gemspec:13]: Fixed.

Comment (cheempz) [inline solarwinds_apm.gemspec:13]: Could you change line 13 `s.homepage` to the placeholder `https://documentation.solarwinds.com/en/success_center/observability/content/intro/landing-page.html` too?

File: solarwinds_apm.gemspec [hunk 2]
```diff
@@ -17,10 +17,10 @@ Automatic tracing and metrics for Ruby applications. Get started at cloud.solarw
   EOF
 
   s.metadata = {
-      'changelog_uri'     => 'https://github.com/appoptics/appoptics-apm-ruby/releases',
-      'documentation_uri' => 'https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=config-ruby-agent',
-      'homepage_uri'      => 'https://cloud.solarwinds.com/',
-      'source_code_uri'   => 'https://github.com/appoptics/appoptics-apm-ruby',
+      'changelog_uri'     => 'https://github.com/solarwindscloud/solarwinds-apm-ruby/releases',
+      'documentation_uri' => 'https://documentation.solarwinds.com/en/success_center/observability/content/intro/landing-page.html',
```
Comment (cheempz) [inline solarwinds_apm.gemspec:21]: The `documentation_uri` was actually correct, please revert to `https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=config-ruby-agent`

Comment (xuan-cao-swi) [reply solarwinds_apm.gemspec]: updated.

File: solarwinds_apm.gemspec (hunk 1 above)
Comment (xuan-cao-swi) [reply solarwinds_apm.gemspec]: updated.

**Review by cheempz — APPROVED** [review]
LGTM, thanks for the revisit @xuan-cao-swi!

**Pull request merged**